src/chainstate/stacks/db/mod.rs

```diff
@@ -265,6 +265,11 @@ impl ClarityConnection for ClarityTx<'_> {
         ClarityConnection::with_clarity_db_readonly(&mut self.block, to_do)
     }
 
+    fn with_clarity_db_readonly_owned<F, R>(&mut self, to_do: F) -> R
```

Comment:
Is this used anywhere?

Reply:
Yes, it's used by the auto-implemented function in the trait.